walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -22,59 +22,86 @@
 
 /** Finds potential meeting times. */
 public final class FindMeetingQuery {
+  public Collection<TimeRange> timesForEveryone;
+  public Collection<TimeRange> timesForConfirmed;
+  public int currentEndTime;
```

Comment:
class members should never be public. If you need to access them then you should create methods to return their values.